config.yaml

```diff
@@ -2,9 +2,7 @@ name: liquidation-metrics # Name of the project
 description: Liquidation data for Compound-V2 # Description of the project
 networks: # Configuration of the blockchain networks that the project is deployed on
   - id: 1 # Chain identifier of the network (ethereum mainnet)
-    rpc_config:
-      url: #http://localhost:8545 # RPC URL that will be used to subscribe to blockchain data on this network
-    start_block: 0 # Initial block from which the indexer will start listening for events
+    start_block: 8983575 # Initial block from which the indexer will start listening for events
```

Comment:
Took the relevant start block from when the contract was deployed, it means the indexer doesnt need to start looking for events from block 0

Reply:
I mentally had a TODO for this but I forgot lol